nova/virt/vmwareapi/vm_util.py

```diff
@@ -1315,7 +1315,7 @@ def update_cluster_placement(session, context, instance, cluster, vm_ref):
         return
 
     with lockutils.lock(server_group_info.uuid,
-                        lock_file_prefix='nova-vmware-server-group'):
+                        lock_file_prefix='vmware-vm-group-policy'):
```

Comment:
I don't think that lock is necessary, since `cluster_util.update_placement()` is guarded by `@utils.synchronized` already.

Reply:
Yes since both end up doing reconfigureCluster this is not needed actually. I'll remove it.